stories/core/Footer.stories.tsx

```diff
@@ -46,6 +46,39 @@ Custom.args = {
   ],
 } as FooterProps;
 
+export const OnlyCustomElements: Story<FooterProps> = ({
+  customElements,
+  showOnlyCustomElements,
+  ...rest
+}: FooterProps) => {
+  return (
+    <Footer
+      customElements={customElements}
```

Comment:
For better code readability in Storybook, having a definition of `customElements` in the story would be helpful.
`const customElements = [...]`

Reply:
Fixed

Reply:
I dont think you did it.
I just mean copying into the story code too like in this story for example
https://github.com/iTwin/iTwinUI-react/blob/c0ad0921f859130f4cc866999ba2d31187b192ac/stories/core/DropdownMenu.stories.tsx#L95
```js
const customElements: [
    {
      title: 'Custom Element 1',
      url: 'https://www.bentley.com/',
    },
    {
      title: 'Custom Element 2',
    },
    {
      title: 'Custom Element 3',
    },
    {
      title: 'Custom Element 4',
    },
  ],

Reply:
Ah, I fixed it this time.